SECURITY.md

```diff
@@ -0,0 +1,13 @@
+# Security Policy
+
+Siemens takes the security of its code seriously. If you think you have found a security vulnerability,
+please read the next sections and follow the instructions to report your finding.
+
+## Reporting a Vulnerability
+
+Please DO NOT report any potential security vulnerability via a public channel (mailing list, GitHub issue etc.).
+Instead, [report the vulnerability privately via GitHub](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability)
+(if enabled for the repository) or [contact us via email](mailto:opensource@siemens.com).
+
+Please provide a detailed description of the issue, the steps to reproduce it, the affected versions and, if already available,
+a proposal for a fix. You should receive a response within 5 working days.
```

Comment:
Did you intentionally drop sentences like "If the issue is confirmed as a vulnerability by us, we will open a Security Advisory on github and give credits for your report if desired." and "This project follows a 90 day disclosure timeline."? The latter is indeed quite concrete but also not overly challenging.

Reply:
I have no further comments, except one. Even though 90 days might not be challenging, it will become a metric against which we will be measured... By leaving it out we have more wiggle room. ("we never promised to do this by then.")

Reply:
@jan-kiszka Yes, I dropped that intentionally because I thought that might be too much to enforce on all maintainers.

But with @nejch suggestion to send a message we could also add it back and tell maintainers to set up their own policy if they want something different.

Reply:
I added back the sentences, but kept the disclosure timeline unspecific. I think this strongly depends on the project and the kind of vulnerability.

WDYT @jan-kiszka?